src/app/downloads/downloads.component.html

```diff
@@ -16,16 +16,15 @@ <h3 class="col-12 mb-2 mt-4 card-subtitle text-muted">Select your platform to st
             <a class="btn btn-default btn-primary btn-block btn-bg-dark mb-1" role="button" [href]="latestRelease.windows_main" (click)="trackDownload('windows', latestRelease.windows_main)">Windows (x64)</a>
           </div>
           <div class="col-xs-12 col-sm-4 col-lg-4 mr-1">
-            <a class="btn btn-default btn-primary btn-block btn-bg-dark mb-1" role="button" [href]="latestRelease.ubuntu_22" (click)="trackDownload('linux', latestRelease.ubuntu_22)">Ubuntu 22.04</a>
+            <a class="btn btn-default btn-primary btn-block btn-bg-dark mb-1" role="button" [href]="latestRelease.ubuntu_24" (click)="trackDownload('linux', latestRelease.ubuntu_24)">Ubuntu 24.04</a>
```

Comment:
- Ubuntu 24.04 moved up to the new primary version
- Pushed Ubuntu 22.04 down as the alternate
- Removed 20.04 entirely